### DISCUSSION TITLE
Old Version of LHC compatible with PHP 5.6.40

### DISCUSSION THREAD

**Discussion by Hamburgo58** [LiveHelperChat/livehelperchat#1800] [General]
Good day!

My host installed PHP Version 5.6.40.
Increase the version of php on servers can technically impossible right now.
So please tell me where I can download a version that is compatible with PHP 5.6.40

Thank you!

**Comment by PeopleInside** [comment]
Hi,
if you are on hosting that use PHP 5.6 you probably have security issues and vulnerability as you can see on https://www.php.net/supported-versions.php

I believe we cannot suggest to use an outdated version of LHC on a vulnerable no more supported PHP version.
If you are not able to self-host LHC you can use the hosted version: https://livehelperchat.com/order/now

**Comment by Hamburgo58** [comment]
What will be your risk to offer downloading older versions of LHC ?

To use older versions of PHP, LHC or others will be my risk in case of sicurity issues.

I will not have a problem to host LHC with PHP 7.x as single application.

My problem are my own applications to update them to PHP 7.x, because for that
case I have to modify the whole handling of database and changing the charset
from iso-8859-1 to UTF-8.

This is not a question of hours or days, this will be need time of month.

I am using LHC for my applications in a intra-net environment with much less
security risk as in the publick internet.

Please offer me a download link to a version of LHC that will be compatible with PHP 5.6.40.

Many thanks.

**Comment by remdex** [comment]
It's really to hard to say. There are command line tools which determine minimum PHP version. You could try to download older versions like 3.x and see which one meets your requirements. I'm pretty sure it will be really old ones. As I did not explicitly tracked when I dropped 5.6x support...

**Comment by Hamburgo58** [comment]
I found an old version (3.24) on my server.

After call "index.php/site_admin/install/install" I got this error window.

"Do not have permission to access module "install" and use "install" function"

What is the reason and how can I fix this ?

![LHC_01](https://user-images.githubusercontent.com/10532029/201048476-6e54811f-0086-434b-9055-9143b18d970a.png)

**Comment by remdex** [comment]
This mean that lhc was already installed... You can try to follow
https://doc.livehelperchat.com/docs/development/move-chat

**Comment by Hamburgo58** [comment]
Thanks, but this workflow has the condition that the database will be exist.

In my case I do not have a database for LHC.

What can I do ?

**Comment by remdex** [comment]
If you have fresh install make sure settings/settings.ini.php file does not exists.
